Modules/Core/Common/include/itkVectorContainer.h

```diff
@@ -561,6 +561,18 @@ class ITK_TEMPLATE_EXPORT VectorContainer
     , VectorType(first, last)
   {}
 };
+
+
+/** Makes a VectorContainer that has a copy of the specified `std::vector`. */
+template <typename TElement>
+auto
+MakeVectorContainer(std::vector<TElement> stdVector)
```

Comment:
@dzenanz Indeed. 👍 

```cpp
    std::vector<int> vector123 = { 1, 2, 3 };
    auto container = itk::MakeVectorContainer(vector123);
    // Now vector123 is still { 1, 2, 3 };
```

Only when a user explicitly does `std::move` on the parameter, it may loose its original data:

```cpp
    std::vector<int> movingVector = { 1, 2, 3 };
    auto container = itk::MakeVectorContainer(std::move(movingVector));
    // Now movingVector is empty.
```